driver-jsdom/index.js

```diff
@@ -5,17 +5,24 @@ const jsdom = require('jsdom');
 exports.mochifyDriver = mochifyDriver;
 
 function mochifyDriver(options = {}) {
-  const stderr = options.stderr || process.stderr;
+  const {
+    stderr = process.stderr,
+    url = 'http://localhost',
+    pretendToBeVisual = true,
+    strictSSL = false,
+    ...jsdom_options
+  } = options;
 
   const virtual_console = new jsdom.VirtualConsole();
   const { window } = new jsdom.JSDOM(
     '<!DOCTYPE html>\n<html><body></body></html>',
     {
-      url: options.url || 'http://localhost',
+      url,
+      pretendToBeVisual,
+      strictSSL,
+      ...jsdom_options,
       virtualConsole: virtual_console,
-      runScripts: 'dangerously',
-      pretendToBeVisual: true,
-      strictSSL: false
+      runScripts: 'dangerously'
```

Comment:
Placing these two mandatory settings down here makes sure that it works, but might trip up people that don't read the documentation. Should this rather be allowed to pass through as well, even if it makes Mochify fail?

Reply:
If we don't put it here, wouldn't everyone have to provide this as an option for the driver to work? All drivers should work out of the box – at least to some extend.

Reply:
No, what I meant is allowing consumers to override these in the first place. Right now, you could pass your own runScripts value, but Mochify would still use its preset.